mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Alog.MixProject do
   def project do
     [
       app: :alog,
-      version: "0.4.2",
+      version: "0.4.3",
```

Comment:
I think this should be 0.5.0 now, as we've changed the behaviour of the module

Reply:
I wasn't sure about the version. For me it's a patch fix as the API hasn't changed and I don't think this PR add a new functionality, I see keeping the `inserted_at` value as a bug fix

>    MAJOR version when you make incompatible API changes,
    MINOR version when you add functionality in a backwards-compatible manner, and
    PATCH version when you make backwards-compatible bug fixes.

but happy to to update to `0.5.0` if you think the logic of the code is different :+1:

Reply:
I see how this could be considered a bug, but I think it changes enough that it could introduce unexpected errors into code that uses it, eg. someone could have been using `inserted_by` to get the most recent items in their database, and now that won't work the same.

I think just to be safe we should bump to `0.5.0`

https://xkcd.com/1172/